src/Components/Trips/trips.test.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { shallow, mount } from "enzyme";
+import { cleanup } from "@testing-library/react";
+
+import Trips from "./Trips";
+import Spinner from "../spinner/spinner";
+import Modal from "../Modal/modal";
+
+afterEach(cleanup);
+
+xtest("should display a header", () => {
```

Comment:
This test wouldn't run right

Reply:
It ran. But i added it after running the test, so as not to keep running it. I should have removed it before pushing to github.